improve style

francescofuggitti opened the pull request:
- change anchors color
- fix displaying issues on bibbase entries
- clean comments

marcofavorito commented:
Basically, this is one of the things is improved, right? Better visualization of Bibbase items

![image](https://user-images.githubusercontent.com/9467617/83756676-acdd8200-a66f-11ea-9beb-e61c4dcdf4dd.png)

francescofuggitti commented:
Yes, I changed the color of anchors to be consistent with the main color of the website. Then, I added margins between the `Group by` btn and the menu btn, also increasing the width of dropdowns
